test/functional/api/window_spec.lua

```diff
@@ -889,6 +889,246 @@ describe('API/win', function()
          }))
       eq(wins_before, meths.list_wins())
     end)
+
+    it('can create a split window', function()
```

Comment:
I would drop the "can" from these test names and just use `'creates a split window'`.

Reply:
Will do, thanks for the reviews!